autode/species/complex.py

```diff
@@ -218,7 +223,23 @@ def calc_repulsion(self, mol_index):
 
         return repulsion
 
-    def __init__(self, *args, name='complex'):
+    def _init_translation(self):
+        """Translate all molecules initially to avoid overlaps"""
+
+        if self.n_molecules < 2:
+            return   # No need to translate 0 or 1 molecule
+
+        # Points on the unit sphere maximally displaced from one another
+        points = get_points_on_sphere(n_points=self.n_molecules)
```

Comment:
Why don't you put the radius here? Would make more sense to me

Reply:
it's a useful property of a molecule(?). is used here https://github.com/duartegroup/gap-train/blob/45268911174b94c39e93604dfeb4fb71a2187c0a/gaptrain/molecules.py#L74

Reply:
no, I mean why don't you use the radius when generating the points, instead of multiplying the points by the radius later

Reply:
because the molecules might be different & have different radii